py/server/deephaven/server/executors.py

```diff
@@ -0,0 +1,56 @@
+#
+# Copyright (c) 2016-2022 Deephaven Data Labs and Patent Pending
+#
+"""
+Support for running operations on JVM server threads, so that they can be given work from python. Initially, there
+are two executors, "serial" and "concurrent". Any task that will take an exclusive UGP lock should use the serial
+executor, otherwise the concurrent executor should be used. In the future there may be a "fast" executor, for use
+when there is no chance of using either lock.
+"""
+
+from typing import Callable, Dict, List
+import jpy
+from deephaven.jcompat import j_runnable
+
+
+_executors: Dict[str, Callable[[Callable[[], None]], None]] = {}
+
+
+def has_named_executor(executor_name: str) -> bool:
```

Comment:
Is it even possible to have 'unnamed' executors? If not, maybe just rename to `has_executor`.